spec/services/idv/profile_maker_spec.rb

```diff
@@ -25,16 +25,16 @@
       pii = subject.pii_attributes
 
       expect(profile).to be_a Profile
-      expect(profile.id).to_not be_nil
-      expect(profile.encrypted_pii).to_not be_nil
-      expect(profile.encrypted_pii).to_not match 'Some'
-      expect(profile.proofing_components).to match proofing_component.as_json
-      expect(profile.active).to eq false
-      expect(profile.deactivation_reason).to be_nil
+      expect(profile.id).to_not eq(nil)
```

Comment:
Is there a reason you prefer `to_not eq(nil)` over `to_not be_nil`?

Reply:
I think they are not exactly equivalent, but I don't remember how. But it is more readable.